### ISSUE TITLE
ms-playwright does not detect tests in test explorer in vscode

### ISSUE THREAD

**Issue by rushi-alation** [microsoft/playwright-vscode#475]
Type: <b>Bug</b>

Install extension with latest vscode and extension

Extension version: 1.1.1
VS Code version: Code 1.89.1 (Universal) (dc96b837cf6bb4af9cd736aa3af08cf8279f7685, 2024-05-07T05:14:24.611Z)
OS version: Darwin arm64 23.3.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|8, 5, 5|
|Memory (System)|32.00GB (1.64GB free)|
|Process Argv|--crash-reporter-id a341c1bf-be8b-40bc-a28a-29c69bb7d082|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
tftest:31042121
vswsl492:30256859
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
chatpanelc:31018788
dsvsc021:30996838
945dj816:31013170
pythoncenvpt:31043157
a69g1124:31046351
pythonprc:31039816
dwnewjupyter:31046869

```

</details>

**Comment by mxschmitt** [comment]
Would it be possible to share your repository structure with us? Which Playwright version are you using? Does this happen when you create a new project with `npm init playwright`?

**Comment by rushi-alation** [comment]
> Would it be possible to share your repository structure with us? Which Playwright version are you using? Does this happen when you create a new project with `npm init playwright`?

It happens on the latest version which was released today for the extension. I reverted back to the old version and do not observe this issue

**Comment by nickwang0808** [comment]
Also reproed, rolling back to prev version fixed it. mine was only able to detect the setup test. non of the other tests were detected

**Comment by Sharaminator** [comment]
Confirming also that reverting back to version 1.0.22, resolved the issue for me.

**Comment by nnson0310** [comment]
I faced this issue too. Should roll back to old version.

**Comment by nnson0310** [comment]
All my test suites are gone. When I tried to run, it shows this issue in the console and can not stop the test from running even when trying to pause it with empty square icon
![Uploading 2024-05-14 09_22_28-Greenshot.jpg…]()

**Comment by rapidroamer** [comment]
me too
@playwright/test : 1.38.1
Playwright Test for VSCode v1.1.1

I am  able to see and run only global setup, but the main tests are not lised the test explorer

![image](https://github.com/microsoft/playwright-vscode/assets/22044320/6de3caa2-5b7c-436a-9388-2c4c24046b3d)

**Comment by mxschmitt** [comment]
We need a reproduction in order to triage this issue further - would someone willing to help us to narrow this issue down?

Do you reproduce this on the latest Playwright version (v1.44 as of today)?

Do you reproduce this on a fresh project? `npm init playwright my-new-pw-project`

**Comment by kosteklvp** [comment]
I think that this is caused, because of the new placement of "Projects" section.
![obraz](https://github.com/microsoft/playwright-vscode/assets/29569674/d9795f6f-290a-47c7-8463-ce22523ba9ff)

In my case, all projects were unselected here, which caused an empty "Test Explorer" view.
![obraz](https://github.com/microsoft/playwright-vscode/assets/29569674/1190085d-11f2-4df3-ab12-b465fc19508c)

**Comment by kempeltemple** [comment]
@debs-obrien I'm intrigued, could you elaborate on the difference between a "normal setup" and "monorepo" style? The implication seems to be that I have a non-standard (monorepo) project configuration since I have checkboxes under the "PROJECTS" heading.

As far as I'm aware, my Playwright config file is set up according to the official documentation, so I'm wondering what I'm missing (or misunderstanding).

**Comment by debs-obrien** [comment]
sorry, by normal setup I mean straight new install. There seems to be a few different issues so just trying to find ways to reproduce each of them. In my example, projects don't even get shown so there is nothing to be selected so its impossible to run. If anyone else has a way of reproducing and sharing please do

**Comment by KocherginV** [comment]
I use two config files (different for mobile and desktop screen resolutions). After update my mobile tests were gone from the test explorer, but applying mobile config manually resolved the issue. 
<img width="1466" alt="Screenshot 2024-05-14 at 14 30 06" src="https://github.com/microsoft/playwright-vscode/assets/23701921/6c4e22fb-683f-4ec4-8c3a-28e93ab1fb6c">

**Comment by uznys** [comment]
Multiple Playwright configs is not supported any more.
Here's an example of two test suites with two config files:
https://github.com/uznys/playwright-vscode-1-1-1-bug

**Comment by tbol-donatas** [comment]
> We need a reproduction in order to triage this issue further - would someone willing to help us to narrow this issue down?
> 
> Do you reproduce this on the latest Playwright version (v1.44 as of today)?
> 
> Do you reproduce this on a fresh project? `npm init playwright my-new-pw-project`

@mxschmitt Yes, still reproducible in Playwright 1.44 on a fresh project. See comment above.

**Comment by Dtesch9** [comment]
I had to downgrade the extension to the previous version in order to have my tests suite back, waiting until it's fixed.

**Comment by R-Gifford** [comment]
Can confirm.  In addition, it seems to be an issue with the `playwright.config.ts` if multiple projects are setup, only the first is seen.  You can change the order of the projects and then the top project `[0]` is the only one reconized.

**Comment by KocherginV** [comment]
For me to fix the issue it was enough to:
1. Toggle Playwright configs
2. Select both configs in the opened dialog
<img width="1461" alt="Screenshot 2024-05-14 at 18 16 34" src="https://github.com/microsoft/playwright-vscode/assets/23701921/428bfd12-86e2-45b6-a1bc-2e4c6672fd25">
<img width="1456" alt="Screenshot 2024-05-14 at 18 18 22" src="https://github.com/microsoft/playwright-vscode/assets/23701921/a4de7958-f569-4918-81a1-34fede325ecf">
